catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/ListMetacardImpl.java

```diff
@@ -18,7 +18,7 @@
 import java.util.Collections;
 
 public class ListMetacardImpl extends MetacardImpl {
-  private static final ListMetacardTypeImpl TYPE = new ListMetacardTypeImpl();
+  public static final ListMetacardTypeImpl TYPE = new ListMetacardTypeImpl();
```

Comment:
The `Metacard.getMetacardType()` method is not applicable here because we have no instance of list metacards where it is used; the type is being used to create new metacards of that type [here](https://github.com/Variadicism/ddf/blob/extend-workspacetransformer/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformations/EmbeddedMetacardsHandler.java#L54), which is why it was made public.